integral_view/views/ntp_management.py

```diff
@@ -98,4 +101,22 @@ def update_ntp_settings(request):
         return django.shortcuts.render_to_response("logged_in_error.html", return_dict, context_instance=django.template.context.RequestContext(request))
 
 
+def sync_ntp(request):
+    return_dict = {}
+    try:
+        output, err = ntp.sync_ntp()
+        if err:
+            raise Exception(err)
+        else:
+            if 'ntp_sync' in output and output['ntp_sync'] == True and 'server_used' in output:
```

Comment:
here i am saying the exception message as - "Couldn't sync because the required parameters doesn't met"